core/src/dird/dird_conf.h

```diff
@@ -131,7 +131,10 @@ class DirectorResource
   bool optimize_for_speed = false; /* Optimize daemon for speed which may need
                               more memory */
   bool ndmp_snooping = false;      /* NDMP Protocol specific snooping enabled */
-  bool auditing = false;           /* Auditing enabled */
+  bool ndmp_fhinfo_set_zero_for_invalid_u_quad
+      = false;  // Workaround for Isilon 9.1.0.0 not accepting -1 as value for
+                // FhInfo (which is the tape offset)
+  bool auditing = false; /* Auditing enabled */
```

Comment:
```suggestion
  bool auditing = false;           /* Auditing enabled */
```

Reply:
As we autoformat our sourcecode with clang-format, this change is not possible to do as it will be reformatted automatically.